MdePkg/Test/Mock/Library/GoogleTest/MockUefiRuntimeServicesTableLib/MockUefiRuntimeServicesTableLib.cpp

```diff
@@ -36,5 +36,5 @@ static EFI_RUNTIME_SERVICES localRt = {
 };
 
 extern "C" {
-  EFI_RUNTIME_SERVICES* gRT = &localRt;
+EFI_RUNTIME_SERVICES  *gRT = &localRt;

```

Comment:
are these formatting changes applied via uncrustify?

Reply:
Yeah - but I don't think i have your latest changes - let me see about pulling them in